src/components/ui/practiceDropdown.jsx

```
@@ -0,0 +1,49 @@
+      const path = {
+         sentenceBbuilder: "/sentence-builder",
+         wordMatching: "/wordmatching",
+         memoryCard: "/memoryCard",
+         storyGame: "/story-game",
+         dragAndDrop: "/drag-and-drop",
+         idiomChallenge: "/idiom-challenge",
+         multipleChoice: "/multiple-choice",
+      }[key.toLowerCase()];
+
+      if (path) {
+         navigate(path);
```

ESLint

[error] 'navigate' is not defined.

(no-undef)